src/components/SceneFeatureDetails.tsx

```diff
@@ -18,32 +18,34 @@ const styles: any = require('./SceneFeatureDetails.css')
 
 import * as React from 'react'
 import * as moment from 'moment'
+import {SCENE_TILE_PROVIDERS} from '../config'
 
 interface Props {
   className?: string
   feature: beachfront.Scene
 }
 
-export const SceneFeatureDetails = ({className, feature}: Props) => (
-  <div className={`${styles.root} ${className || ''}`}>
-    <h1 title={normalizeId(feature.id)}>{normalizeId(feature.id)}</h1>
+export const SceneFeatureDetails = ({className, feature}: Props) => {
+  const id = normalizeSceneId(feature.id)
+
+  return <div className={`${styles.root} ${className || ''}`}>
+    <h1 title={id}>{id}</h1>
 
     <dl>
       <dt>Date Captured</dt>
-      <dd>{moment(feature.properties.acquiredDate).utc().format('MM/DD/YYYY HH:mm z')}</dd>
+      <dd>{moment.utc(feature.properties.acquiredDate).format('MM/DD/YYYY HH:mm z')}</dd>
 
       <dt>Cloud Cover</dt>
-      <dd>{Math.round(feature.properties.cloudCover)}%</dd>
+      <dd>{feature.properties.cloudCover.toFixed(0)}%</dd>
 
       <dt>Sensor Name</dt>
       <dd>{feature.properties.sensorName}</dd>
     </dl>
   </div>
-)
+}
+
+const stripProviderRe = new RegExp(`^(${SCENE_TILE_PROVIDERS.map(p => p.prefix).join('|')}):`)
 
-function normalizeId(featureId) {
-  if (!featureId) {
-    return 'nil'
-  }
-  return featureId.replace(/^(landsat|sentinel|rapideye|planetscope):/, '')
```

Comment:
Is there any other place these string literals are used that can be removed?  We don't want this source list hanging out everywhere so we have to update a bunch of places whenever a new source is added

Reply:
I think I caught all of them.